Sigma_specification.md

```diff
@@ -592,10 +576,11 @@ A Sigma rule can be categorised with tags. Tags should generally follow this syn
 * Keep tags short, e.g. numeric identifiers instead of long sentences
 * Feel free to send pull request or issues with proposals for new tags
 
-[More information about tags](appendix_tags.md)
+[More information about tags](appendix/appendix_tags.md)
 
 ## Placeholders
-
+/// NEED REVIEW
+///
```

Comment:
to be removed

Reply:
Even if Placeholders section look goot to me , is it good ?